src/main/resources/assets/tconstruct/book/materials_and_you/en_us/modifiers/reinforced.json

```diff
@@ -0,0 +1,13 @@
+{
+  "modifier_id": "tconstruct:reinforced",
+  "text": [
+    {
+      "text": "Strengthening your tool might be less work for both of you!"
+    }
+  ],
+  "effects": [
+    "Makes the tool last longer",
+    "Multiple levels, each level increases the chance for durability to be consumed",
```

Comment:
Chance for durability to not be consumed. Probably more clear to say each level decreases the chance of the tool taking damage